project-base/storefront/components/Blocks/Product/Filter/FilterSelectedParameters.tsx

```
@@ -29,173 +31,203 @@
+                                                        }
+                                                    >
+                                                        {selectedValue.rgbHex && selectedValue.rgbHex !== '' && (
+                                                            <div
+                                                                className="mr-2 h-4 w-4 rounded border border-text"
+                                                                style={{ backgroundColor: selectedValue.rgbHex }}
+                                                            />
+                                                        )}
+                                                        {selectedValue.text}
+                                                        <SelectedParametersIcon />
+                                                    </SelectedParametersListItem>
+                                                ))}
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)